src/sentry/snuba/tasks.py

```diff
@@ -188,22 +189,30 @@ def build_query_builder(
 
 
 def _create_in_snuba(subscription: QuerySubscription) -> str:
-    snuba_query = subscription.snuba_query
-    entity_subscription = get_entity_subscription_from_snuba_query(
-        snuba_query,
-        subscription.project.organization_id,
-    )
-    snql_query = build_query_builder(
-        entity_subscription,
-        snuba_query.query,
-        [subscription.project_id],
-        snuba_query.environment,
-        params={
-            "organization_id": subscription.project.organization_id,
-            "project_id": [subscription.project_id],
-        },
-    ).get_snql_query()
-    return _create_snql_in_snuba(subscription, snuba_query, snql_query, entity_subscription)
+    with sentry_sdk.start_span(op="create_in_snuba") as span:
+        span.set_data(
+            "uses_metrics_layer",
```

Comment:
Should we use a tag here instead so we can search for transactions using the metrics layer?

Reply:
Sure!